source/administration/config-settings.rst

```diff
@@ -2745,6 +2745,29 @@ To manage who can create personal access tokens or to search users by token ID,
 | This feature's ``config.json`` setting is ``"EnableUserAccessTokens": false`` with options ``true`` and ``false``.                                                   |
 +----------------------------------------------------------------------------------------------------------------------------------------------------------------------+
 
+
+Bot Accounts
+~~~~~~~~~~~~~~~~~~~~~~~~~
+Enable Bot Account Creation
```

Comment:
@aaronrothschild Should this setting be removed from "Integrations" section above or does it still also exist there in the system console?

Reply:
On 5.14 it no longer appears in the "Integrations" page

Reply:
Cool, then it can be removed from that section.